kyc/views/res_partner_view.xml

```diff
@@ -122,113 +116,106 @@
                                     type="object"
                                     string="Refresh KYC Ongoing Monitoring"
                                     class="btn btn-link"
-                                    attrs="{'invisible': ['|', ('kyc_status', 'in', ['pending', 'error']), ('kyc_ongoing_monitoring', '=', False)]}"
+                                    invisible="kyc_status in ['pending', 'error'] or not kyc_ongoing_monitoring"
                                 />
                             </div>
                             <field name="kyc_is_expired" invisible="1" />
                             <field name="kyc_is_about_expire" invisible="1" />
                         </group>
                     </group>
-                    <group name="kyc_documents" string="Documents">
-                        <field name="kyc_company_passport_required" invisible="1" />
-                        <div>
-                            <p
-                                class="oe_grey"
-                                attrs="{'invisible':['|', ('is_company','=',True), ('is_government','=',True)]}"
-                            >
-                                Required Document: Passport or ID card
-                            </p>
-                            <p
-                                class="oe_grey"
-                                attrs="{'invisible':['|', ('is_company','=',False), ('is_government','=',True)]}"
-                            >
-                                Required Documents:
-                                <ul>
-                                    <li
-                                        attrs="{'invisible':[('kyc_company_passport_required','=',False)]}"
-                                    >Passport for each Ultimate Beneficial Owner.</li>
-                                    <li
-                                    >An owner certificate (a single one for all UBOs)</li>
-                                </ul>
 
-                            </p>
-                            <span class="oe_inline">
+                    <seprator string="Documents" />
```

Comment:
seprator ?

Reply:
![image](https://github.com/user-attachments/assets/6fb51b4e-d5c0-4530-907b-6451e990b3b5)
For this part of view